### ISSUE TITLE
Peer Review - for UH-A1 include what failed

### ISSUE THREAD

**Issue by bahsounm** [emilyperica/ScoreGen#293]
**Artifact Under Review**
VnV Report
**Team Number for Team Doing the Review**
Team - 8
**Description of Issue**
Test UH-A1 (WCAG Compliance) fails but does not specify which WCAG 2.1 failed, I would say it would be helpful to include them for future reference so you have track of what worked and what did not
